package.json

```diff
@@ -29,6 +29,9 @@
     "@next/third-parties": "^14.2.3",
     "@prisma/client": "^5.13.0",
     "@sanity/diff-match-patch": "^3.1.1",
+    "@supabase/auth-ui-react": "^0.4.7",
```

Comment:
@enricoros I wasn't able to get the supabase auth UI working nicely, I think these might be able to be removed, thoughts?

Reply:
I'll take a look at the default components provided by the package, I shall be able to integrate this. Worst case we drop this package and code a manual UI.